Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM ubuntu:latest
+ENV PATH $PATH:/usr/local/bin
+ENV TERRAFORM_VER 0.6.7
+ENV TERRAFORM_ZIP terraform_"$TERRAFORM_VER"_linux_amd64.zip
+
+RUN apt-get update && apt-get -y install wget unzip openssh-client dnsutils git pass ruby
```

Comment:
Ok, fair enough. Pass, git, dnsutils are removed now and commit message modified.